types/event.model.ts

```diff
@@ -0,0 +1,40 @@
+import { Crew, Team } from './crew.model';
+import { DisplayStatus } from './display-status.model';
+import { Round } from './round.model';
+
+export interface Event {
+	id: string;
+	regatta_id: string;
+	cost_id: string;
+	parent: null; // Wat is de 'parent'?
+	event: number;
+	number: number; // Is dit hetzelfde als 'event'?
+	day: Date; // Wordt een event op alleen een dag gedaan? Is dat anders dan de regatta?
+	code: string; 
+	name: string; // Is dit hetzelfde als 'name'?
+	remarks: string;
+	status: DisplayStatus; // Wat zijn de verschillende statussen?
+	category: string;
+	block_knrb: null; // Wat is dit?
```

Comment:
Dit is nummer|null
Eigenlijk nooit ingesteld maar soms word een event al ingedeeld in blok 2 bijvoorbeeld

Reply:
Kan het dan ook een id zijn ipv een nummer van een block?

Reply:
Nee dit is helaas geen id alleen een nummer zonder verdere info

block 1, block 2 ect.